ui/app/components/ui/page-container/page-container-footer/page-container-footer.component.js

```diff
@@ -61,10 +61,10 @@ export default class PageContainerFooter extends Component {
           >
             { submitText || this.context.t('next') }
           </Button>
-        </header>
+        </footer>
 
         {children && (
-          <footer>
+          <footer className="page-container__footer-secondary">
```

Comment:
This is a weird case and it's only used in a single place (for the "Reject all" button on batch transactions). This was the simplest path to ensuring the CSS remained correct. It's weird having multiple footers, but we'll be scrapping this relatively soon with the new batch transaction designs anyway.

Reply:
Darn, these styles aren't being applied because of CSS specificity :disappointed: I'll find a workaround.

Reply:
This has been resolved - no double-footer, and the styles are applied correctly.